frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1246,6 +1281,10 @@ pub mod pallet {
 	#[pallet::getter(fn round)]
 	pub type Round<T: Config> = StorageValue<_, u32, ValueQuery, DefaultForRound>;
 
+	/// Counter of the number of signed and unsigned blocks since the start of the current round.
+	#[pallet::storage]
+	pub type ElectionBlocksCount<T: Config> = StorageValue<_, BlockNumberFor<T>, ValueQuery>;
```

Comment:
Perhaps we can just store the block number when the election started and calculate this dynamically?

Reply:
Replied in the comment above